### PR TITLE
Adding  advice on styling figures

### PR DESCRIPTION
**Pull request by VLucet**
This is to help with #437

### CODE REVIEW

File: docs/syntax/markdown.md [hunk 1]
```diff
@@ -345,6 +345,43 @@ In fact the syntax `\fig{./test}` is also allowed, Franklin will then first look
...
 @@
 
+Figure style (for instance, if one wishes to modify figure width) can be further customized in two different ways. A first way is to define custom CSS for the figure. In the markdown, this comes down to writing:
+
+```
+@@im-50
+![](/assets/rndimg.jpg)
+@@
+```
+
```
Comment (tlienart) [inline docs/syntax/markdown.md:355]: maybe here add "which puts the image in a div block with a class name `im-50`"

**Review by tlienart — APPROVED** [review]
This is great, thank you! minor comments to address and it should be good to go (ignore broken CI)

**Comment by tlienart** [comment]
@VLucet small bump, are you happy to do the changes before it gets merged? otherwise I can take over the PR and do these changes myself. Thanks!

**Comment by VLucet** [comment]
Sorry for the delay, and thank you 🙏🏼 for your patience. That should do it 😃

**Comment by tlienart** [comment]
Thanks a lot!

**Pull request merged**